roles/README.md

```diff
@@ -0,0 +1,46 @@
+<h1 align="center">
+  Stratum Roles
```

Comment:
SRI Roles
```suggestion
  Stratum Roles
```

Reply:
@GitGab19 what was the suggestion here? it's showing like this:
```diff
-  Stratum Roles
+  Stratum Roles
```

which was maybe a mistake, since it's not changing anything at all?

Reply:
Updated, thanks @plebhash